src/main/java/com/synopsys/integration/detect/tool/detector/DetectorRuleFactory.java

```diff
@@ -119,6 +120,7 @@ private DetectorRuleSet createRules(final DetectDetectableFactory detectableFact
 
         final DetectorRule pipEnv = ruleSet.addDetector(DetectorType.PIP, "Pip Env", PipenvDetectable.class, detectableFactory::createPipenvDetectable).defaults().build();
         final DetectorRule pipInspector = ruleSet.addDetector(DetectorType.PIP, "Pip Inspector", PipInspectorDetectable.class, detectableFactory::createPipInspectorDetectable).defaults().build();
+        final DetectorRule poetry = ruleSet.addDetector(DetectorType.PIP, "Poetry", PoetryDetectable.class, detectableFactory::createPoetryDetectable).defaults().build();
```

Comment:
Poetry uses unique files to queue off of. Since this is a python project, is there a change the other Python detectors are triggered to extract as well such as a requirements.txt?

Does it make sense to yield the other python detectors to poetry?

Reply:
"Does it make sense to yield the other python detectors to poetry?"
>> That's a good question.  Not really sure, to be honest, but I feel like if a pyproject.toml file is found then the project should be treated exclusively as a poetry project, as no other package managers would have that file